xml/depl_cisco_opflex.xml

```diff
@@ -8,364 +8,411 @@
 <appendix xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="app.deploy.cisco_opflex">
  <title>Cisco ACI Integration with Opflex ML2 and GBP Drivers</title>
  <info>
-<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
-    <dm:maintainer>fs</dm:maintainer>
-    <dm:status>editing</dm:status>
-    <dm:deadline/>
-    <dm:priority/>
-    <dm:translation>no</dm:translation>
-    <dm:languages/>
-</dm:docmanager>
-</info>
-<para/>
-<sect1 xml:id="sec.deployment">
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:maintainer>cs</dm:maintainer>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>no</dm:translation>
+   <dm:languages/>
+  </dm:docmanager>
+ </info>
+ <sect1 xml:id="sec.deployment">
   <title>Deployment Into a Cisco ACI Fabric</title>
 
   <para>
- SUSE customers who wish to deploy &cloud; into a Cisco Application Centric Infrastructure (ACI) fabric environment can do this by using Cisco OpFlex ML2 or GBP drivers. Cisco ACI options are selected in the Neutron barclamp, which will make your &cloud; installation a member of your Cisco Application Policy Infrastructure Controller (APIC). The drivers are packaged as RPMs for easy installation.  <xref linkend="sec.barclamps"/> details how to use the ML2 drivers, and <xref linkend="sec.gbp"/> has the steps for using the GBP drivers.
- </para>
-   <para>
- This is a technology preview. We have successfully tested these steps on systems directly connected to a switch port on the ACI fabric. After you have set this up you should be able to:
+   SUSE customers who wish to deploy &cloud; into a Cisco Application Centric
```

Comment:
We tend not to distance ourselves to much from our customers, hence: "SUSE customers who wish..." -> "If you wish..."